src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -190,6 +295,7 @@ private bool ApplicableOperatorIs(MethodSymbol candidate, CSharpSyntaxNode node,
         private BoundPattern BindDeclarationPattern(
             DeclarationPatternSyntax node,
             BoundExpression operand,
+            bool? operandIsNull,
```

Comment:
The ```operandIsNull``` parameter is not used. Consider removing, or, if you are planning to use it going forward, consider adding PROTOTYPE comment for it.